src/prommis/nanofiltration/costing/diafiltration_cost_model.py

```diff
@@ -29,72 +37,47 @@ def build_global_params(self):
         # Register currency and conversion rates based on CE Index
         register_idaes_currency_units()
 
-        # initialize the common global parameters
-        self._build_common_global_params()
-
         # Set the base year for all costs
         self.base_currency = units.USD_2021
 
         # Set a base period for all operating costs
         self.base_period = units.year
 
-        # the following global parameters are from the reference file
-        self.factor_total_investment = Var(
+        # fix the parameters
+        self.fix_all_vars()
```

Comment:
What does this call do?

Reply:
Actually, this is not needed, since I removed all other global parameter declarations in this method.